### PR TITLE
Fix humphd/brackets#129 - HintHinter Gutter too large, and bubble looks odd in Large Text Mode

### PR DESCRIPTION
**Pull request by AJDBenner**
Fixes https://github.com/humphd/brackets/issues/129.
Changes look like this

Small

screen shot 2015-03-25 at 2 48 07 pm

Medium

screen shot 2015-03-25 at 2 40 46 pm

Large

screen shot 2015-03-25 at 2 49 05 pm

### CODE REVIEW

**Comment by puzzledShark** [comment]
hmm, the highlight looks weird, have you just tried removing all padding to the extra gutter we add?
In our previous version we had it set to like 1px

**Comment by AJDBenner** [comment]
I added a little padding to the left of the button and removed left and right padding on the line number gutter.

**Comment by AJDBenner** [comment]
The highlight is underneath the button when the font is on large.

screen shot 2015-03-25 at 3 21 04 pm

File: main.less [hunk 1]
```diff
@@ -1,32 +1,25 @@
+/*Font family for errorPanel*/
+@fontFamily: Helvetica, Arial, "Meiryo UI", "ＭＳ Ｐゴシック", "MS PGothic", sans-serif;
```
Comment (gideonthomas) [inline main.less:2]: Why is this a font `ＭＳ Ｐゴシック` :P

File: main.less [hunk 2]
```diff
@@ -1,32 +1,25 @@
...
 .errorHighlight {
     background-color: #EDC5C5;
 }
 
-.errorButton {
-    margin: 1px;
-    background-color: #FF0000;
-}
-
-.errorText {
```
Comment (gideonthomas) [inline main.less:10]: pretty sure this needs to be there. It styles the text in the box containing error details.

**Comment by AJDBenner** [comment]
Updated screenshot on font large.

screen shot 2015-03-25 at 3 29 53 pm

**Comment by puzzledShark** [comment]
That certainly looks better, but I wonder if it'd just be better to look into copying thimble, maybe we should go ask some designers what they think, maybe a little early for that, oh well, good work.

File: main.less (hunk 1 above)
Comment (AJDBenner) [inline main.less:2]: I kept the fonts consistent with [this](https://github.com/humphd/brackets/blob/bramble/src/styles/brackets_fonts.less#L83). Do you think I should remove it?

**Comment by gideonthomas** [comment]
Don't think its a priority right now. But will definitely happen soon.
@AJDBenner looks good to me.

**Comment by puzzledShark** [comment]
It doesn't fix the Gutter being too large, but that seems that may be an even bigger issue, you've fixed the Odd text issue though so, r++

File: main.less (hunk 1 above)
Comment (gideonthomas) [inline main.less:2]: Ok you can keep it

**Comment by AJDBenner** [comment]
I don't think we can make the gutter any smaller without risking cutting off line numbers. This patch sets the line number gutter padding left and right to 0px. I'm not sure if brackets will allow the line number gutter to be more skinny.

**Comment by gideonthomas** [comment]
r=me

**Comment by puzzledShark** [comment]
If you checkout the old version of thimble it automatically scales the gutter for line numbers, I'm somewhat wondering why adobe decided to set the gutter for line numbers to be so large, makes me think they may have used it for something else. But like I said, this is good enough for now, r++

**Comment by AJDBenner** [comment]
I see what you're saying, the old version of thimble has a really skinny gutter compared to this one.

**Comment by puzzledShark** [comment]
I'm only really bringing up conjecture, I'll investigate this later, good work, gonna merge.

**Pull request merged**